siswrap/wrapper_services.py

```diff
@@ -174,8 +200,7 @@ def run(self):
                 proc = subprocess.Popen(["/bin/sleep", "1m"])
                 exec_string = "/bin/sleep 1m"
             else:
-                exec_string = ExecString(self, self.conf_svc,
-                                         self.info.runfolder).text
+                exec_string = self.get_exec_string()
```

Comment:
One can tell you're coming from the Java world will all those getters and setters. :)